Client/reducers/reducer_filtered_restaurants.js

```diff
@@ -0,0 +1,10 @@
+import { PANEL_CLICK_FILTER_ITEM } from '../actions/index';
+
+export default function(state = {}, action) {
+  switch (action.type) {
+  case PANEL_CLICK_FILTER_ITEM:
+    return {...state, filterSelected: [action.payload.selectedFilters]};
+  case PANEL_CLICK_FILTER_ITEM:
```

Comment:
These two cases should be rounded up into the same case since they are in the same file
return {
  ...state,
  filterSelected: action.payload.selectedFilters,
  filteredRestaurants: action.payload.filteredRestaurants
}